### PR TITLE
Marketing Work Requests

### PR DESCRIPTION
**Pull request by mpigsley**

### CODE REVIEW

File: src/cms/testimonials/lexie-egerer_2021-10-28-15-54-06.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ quote: "LifeLoop is an amazing community resource for First Place- Phoenix. The
   to come. First Place Staff can track both individual resident progress and
   monitor community trends to stay up to date on how to best support the
   community’s needs. LifeLoop has helped turn an apartment building into a
-  community we can all feel connected in. "
+  community we can all feel connected in. '
```
Comment (NotoriousLisp) [inline src/cms/testimonials/lexie-egerer_2021-10-28-15-54-06.md:12]: Trailing space

File: src/cms/training-faqs/can-i-cancel-a-request.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-question: Can I cancel a request?
+question: Can I cancel a Request?
 answer: >-
   Yes, you are able to cancel a request. To cancel select ''View Details' on the
```
Comment (NotoriousLisp) [inline src/cms/training-faqs/can-i-cancel-a-request.md:4]: Double single quotes here on purpose?

File: src/cms/training-faqs/i-dont-see-work-requests-when-i-login-where-do-i-find-it.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+question: I don't see Work Requests when I login, where do I find it?
+answer: -Once logged in you can find the Work Requests tab on the left-hand side
+  of the screen in the blue LifeLoop task bar. Select and your requests view
```
Comment (NotoriousLisp) [inline src/cms/training-faqs/i-dont-see-work-requests-when-i-login-where-do-i-find-it.md:4]: `Select` what?

**Review by NotoriousLisp — APPROVED** [review]

**Comment by mpigsley** [comment]
@NotoriousLisp the formatting on the Q&A stuff you commented on is what CS has done thus far. I'll leave it up to them to edit and change as they see fit.

**Pull request merged**